src/pages/home/ReportScreen.js

```diff
@@ -128,7 +128,7 @@ const defaultProps = {
  */
 function getReportID(route) {
     // // The reportID is used inside a collection key and should not be empty, as an empty reportID will result in the entire collection being returned.
```

Comment:
```suggestion
    // The reportID is used inside a collection key and should not be empty, as an empty reportID will result in the entire collection being returned.
```

This has been driving me nuts for weeks 😂 mind fixing it? It's also great that the `null` default value completely went against the warning in the comment.

Reply:
Oh, also... you could maybe update the comment to explain why the `0` is outside of `lodashGet()` instead of using it as the default value of `lodashGet`. In fact, maybe I missed that part... why isn't the code like this?

```
lodashGet(route, 'params.reportID', 0)
```

Reply:
Ah yes, that's right. Can you please add that to the comment so that others will understand that too? It's very subtle.

Reply:
Will do